website/src/controllers/api-v2/groups.js

```diff
@@ -691,8 +691,9 @@ var inviteByEmails = function(invites, group, req, res, next){
 
 api.invite = function(req, res, next){
   var group = res.locals.group;
-
-  if (req.body.uuids) {
+  if (group.memberCount + group.invites.length >= 30) {
+    return res.json(400, {err: "Over 30 member party cap"});
```

Comment:
Oh, I thought this was a different line.  I was asking about doing the translation in `invite-friends.jade`.  As for the 400 error message in `groups.js` I don't know how that's done.  I just copied that from line 702 which is also not translated.